charts/rekor-tiles/templates/_helpers.tpl

```diff
@@ -0,0 +1,185 @@
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "rekor-tiles.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "rekor-tiles.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "rekor-tiles.labels" -}}
+helm.sh/chart: {{ include "rekor-tiles.chart" . }}
+{{ include "rekor-tiles.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "rekor-tiles.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "rekor-tiles.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "rekor-tiles.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "rekor-tiles.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+{{/*
+Server Arguments
+*/}}
+{{- define "rekor-tiles.args" -}}
+- "rekor-server"
+- "serve"
+{{- if .Values.antispam }}
+{{- if .Values.antispam.persistent }}
+- "--persistent-antispam"
+{{- end }}
+{{- if .Values.antispam.maxBatchSize }}
+- {{ printf "--antispam-max-batch-size=%d" (.Values.antispam.maxBatchSize | int) | quote }}
+{{- end }}
+{{- if .Values.antispam.pushbackThreshold }}
+- {{ printf "--antispam-pushback-threshold=%d" (.Values.antispam.pushbackThreshold | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.tesseraLifecycle }}
+{{- if .Values.tesseraLifecycle.batchMaxAge }}
+- {{ printf "--batch-max-age=%s" .Values.tesseraLifecycle.batchMaxAge | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.batchMaxSize }}
+- {{ printf "--batch-max-size=%d" (.Values.tesseraLifecycle.batchMaxSize | int) | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.checkpointInterval }}
+- {{ printf "--checkpoint-interval=%s" .Values.tesseraLifecycle.checkpointInterval | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.pushbackMaxOutstanding }}
+- {{ printf "--pushback-max-outstanding=%d" (.Values.tesseraLifecycle.pushbackMaxOutstanding | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.clientSigningAlgorithms }}
+- {{ printf "--client-signing-algorithms=%s" .Values.clientSigningAlgorithms | quote }}
+{{- end }}
+- {{ printf "--gcp-bucket=%s" .Values.gcp.bucket | quote }}
```

Comment:
Should we check `if .Values.gcp` for a future iteration of supporting other backends?

Reply:
Right now there's no other option so it is a hard requirement, I'd say we can be more flexible when we have other backends to support